src/main/java/lithium/university/services/TwitterServices.java

```diff
@@ -0,0 +1,69 @@
+package lithium.university.services;
+
+import lithium.university.TwitterProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import twitter4j.Paging;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+import java.util.List;
+
+public class TwitterServices {
+    private static final TwitterServices INSTANCE = new TwitterServices();
+    private final Logger logger = LoggerFactory.getLogger(TwitterServices.class);
+    private TwitterProperties twitterProperties;
+
+    private TwitterServices() {}
+
+    public static TwitterServices getInstance() { return INSTANCE; }
+
+    /*
+     * Takes a message and error checks it before attempting to post to user's twitter
+     * Input: Twitter twitter - twitter instance, String message - message to be posted, int tweetTotal - total limited characters
+     * Output: status object of updated status if successful update, null otherwise
+     * */
+    public Status postToTwitter(Twitter twitter, String message, int tweetTotal) throws TwitterException {
```

Comment:
i like the change.  Now what happens if someone else was to pass in a long tweet vs empty tweet?